src/Trinity.TSL/Trinity.TSL.CodeGen/Cell.cs.cpp

```diff
@@ -211,7 +211,7 @@ source->append(R"::( to its
         /// struct equivalent. A return value indicates whether the 
         /// operation succeeded.
         /// </summary>
-        /// <param name="input>A string to convert.</param>
+        /// <param name="input">A string to convert.</param>
```

Comment:
nice catch.. did you also update the source template? it's in Trinity.TSL/Trinity.TSL.CodeTemplates

Reply:
i will commit later

Reply:
Done